src/components/OptionsSelector.js

```diff
@@ -185,7 +202,7 @@ class OptionsSelector extends Component {
                 <OptionsList
                     ref={el => this.list = el}
                     optionHoveredStyle={styles.hoveredComponentBG}
-                    onSelectRow={this.props.onSelectRow}
```

Comment:
There is another usage here we might want to also focus with the new method

https://github.com/Expensify/Expensify.cash/blob/faddfc45779ecb93aa58ffad725f655d4b571f51/src/components/OptionsSelector.js#L130

Reply:
Oh lol I thought I added that in there, good catch.